src/Frontend-GNOME/MainWindow.cs

```diff
@@ -480,7 +480,7 @@ protected virtual void OnFocusOutEvent(object sender, EventArgs e)
                     return;
                 }
 
-                chatView.OutputMessageTextView.UpdateMarkerline();
+                chatView.UpdateLastSeen();
```

Comment:
This call needs to be in a background thread. See https://github.com/meebey/smuxi/blob/master/src/Frontend-GNOME/Views/Chats/ChatView.cs#L585